python/python/tests/test_async_client.py

```diff
@@ -7796,6 +7796,96 @@ async def test_standalone_client_random_key(self, redis_client: GlideClient):
         # DB 0 should still have no keys, so random_key should still return None
         assert await redis_client.random_key() is None
 
+    @pytest.mark.parametrize("cluster_mode", [False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_dump_restore(self, redis_client: GlideClient):
```

Comment:
```suggestion
    @pytest.mark.parametrize("cluster_mode", [True, False])
    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
    async def test_dump_restore(self, redis_client: TGlideClient):
```
We need to test with cluster and standalone both